jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -695,17 +688,7 @@ private InterpreterResult executeSql(String propertyKey, String sql,
     }
 
     try {
-      List<String> sqlArray;
-      sql = sql.trim();
-      if (splitQuery) {
-        sqlArray = sqlSplitter.splitSql(sql);
-      } else {
-        if (sql.endsWith(";")) {
-          sql = sql.substring(0, sql.length() - 1);
-        }
-        sqlArray = Arrays.asList(sql);
-      }
-
+      List<String> sqlArray = sqlSplitter.splitSql(sql);
       for (int i = 0; i < sqlArray.size(); i++) {
```

Comment:
a [For-Each Loop](https://docs.oracle.com/javase/8/docs/technotes/guides/language/foreach.html) would be nicer

Reply:
addressed